app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe):
+    masks_to_buy = 0
+
+    for friend in friends:
+
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    return f"Friends should buy {masks_to_buy} masks" \
+        if masks_to_buy \
```

Comment:
This condition works, but it'd be better to add explicit comparison with zero to increase code readability.
Also, using regular `if`s might be better than the ternary operator here.

Reply:
Thanks, I'll fix it